src/EnergyPlus/WindowManager.cc

```diff
@@ -6942,8 +7179,29 @@ namespace WindowManager {
             }
         }
 
-        NominalConductance = 1.0 / (rOut + Rbare + rIn); // Autodesk:Uninit Rbare was uninitialized if ngllayer > 4
+        NominalConductance = 1.0 / (rOut + Rbare + rIn);
+        Real64 inputU = state.dataMaterial->Material(state.dataConstruction->Construct(ConstrNum).LayerPoint(1)).SimpleWindowUfactor;
+        // only compute adjustment ratio when there is valid user input U
+        if ((WinterSummerFlag == 1)) {
+            if (inputU > 0) {
+                state.dataHeatBal->CoeffAdjRatio(ConstrNum) = inputU / NominalConductance;
+                NominalConductance = NominalConductance * state.dataHeatBal->CoeffAdjRatio(ConstrNum);
```

Comment:
This line is simply equivalent to: 
```
NominalConductance = inputU;
```
No additional multiplication is need.

Reply:
You're right, I'll simplify this.